tests/qemu/user.pm

```diff
@@ -15,14 +15,20 @@ use warnings;
 use base "consoletest";
 use testapi;
 use utils;
-
+use transactional qw(trup_call check_reboot_changes);
+use version_utils qw(is_transactional);
 
 sub run {
     select_console 'root-console';
 
-    zypper_call 'in qemu-linux-user';
+    if (is_transactional) {
+        trup_call("pkg install qemu-linux-user");
```

Comment:
I was just checking on SLE Micro 5.1, and that package is not available..

```
localhost:~ # transactional-update pkg install qemu-linux-user
Checking for newer version.
transactional-update 3.3.0 started
Options: pkg install qemu-linux-user
Separate /var detected.
2021-07-20 15:25:56 tukit 3.3.0 started
2021-07-20 15:25:56 Options: -c4 open 
2021-07-20 15:25:56 Using snapshot 4 as base for new snapshot 5.
2021-07-20 15:25:56 Syncing /etc of previous snapshot 3 as base into new snapshot "/.snapshots/5/snapshot"
ID: 5
2021-07-20 15:25:56 Transaction completed.
Calling zypper install
2021-07-20 15:25:57 tukit 3.3.0 started
2021-07-20 15:25:57 Options: callext 5 zypper -R {} install qemu-linux-user 
2021-07-20 15:25:57 Executing `zypper -R /tmp/transactional-update-cejbyX install qemu-linux-user`:
Refreshing service 'SUSE_Linux_Enterprise_Micro_5.1_x86_64'.
Loading repository data...
Reading installed packages...
Package 'qemu-linux-user' not found.
Resolving package dependencies...
Nothing to do.
2021-07-20 15:25:57 Application returned with exit status 104.
ERROR: zypper install on /.snapshots/5/snapshot failed with exit code 104!
Removing snapshot #5...
2021-07-20 15:25:58 tukit 3.3.0 started
2021-07-20 15:25:58 Options: abort 5 
2021-07-20 15:25:58 Discarding snapshot 5.
2021-07-20 15:25:58 Transaction completed.
transactional-update finished
```

Reply:
Yes, it is only a test case for oS

Reply:
Ah ok, just didn't see that it was missing in the sle schedule :D

Reply:
It might have been there at the beginning though, sorry if that got more confusing than necessary :D